src/any_llm/providers/otari/otari.py

```
@@ -133,48 +157,54 @@ def _normalize_placeholder_api_key(api_key: str | None) -> str | None:
                 msg = (
                     "Platform mode requires a user token "
-                    f"(pass api_key or set {OTARI_PLATFORM_TOKEN_ENV}/{GATEWAY_PLATFORM_TOKEN_ENV})"
+                    f"(pass api_key or set {OTARI_AI_TOKEN_ENV}/{GATEWAY_PLATFORM_TOKEN_ENV})"
                 )
                 raise ValueError(msg)
             client_kwargs["platform_token"] = token
+            using_platform_token = True
         elif platform_mode is False:
             client_kwargs["api_key"] = resolved_api_key
         else:
             if normalized_api_key:
```

Ruff (0.15.15)

[warning] Use `elif` instead of `else` then `if`, to reduce indentation

Convert to `elif`

(PLR5501)